tests/unit/test_handlers.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from awscli.testutils import unittest, mock
+
+from awscli.handlers import awscli_initialize
+
+
+class TestNoDryRunDocCustomization(unittest.TestCase):
+    def test_no_dry_run_doc_customizations(self):
+        event_emitter = mock.Mock()
+        awscli_initialize(event_emitter)
+        registered_handlers = event_emitter.register.call_args_list
+        # Make sure there is no doc customization for ec2's DryRun
+        self.assertNotIn(
+            mock.call('doc-option-example.ec2.*.dry-run', mock.ANY),
```

Comment:
@jamesls s I cannot think of a better idea. If you have suggestions that would awesome. The two other choices that I considered was:

* Add a functional/integration test to make sure the documentation was not repeated. However, this would rely on the actual wordings in the documentation and I really do not like relying on documentation values that comes from models because if you update the model, it may cause a false positive test failure if the documentation changes.
* Not add a test at all. I think this actually makes the most sense because it seems sort of backwards that a test is added to make sure that a customization was removed. Especially since the --dry-run parameter falls under the same logic as all model driven parameters/commands now. I did not choose this option on the premise of following the principle that we always add tests for PRs.

Let me know if you like either of these ideas.